### ISSUE TITLE
Hardware tracer mode emits full debugging info.

### ISSUE THREAD

**Issue by vext01** [softdevteam/ykrustc#45]
~~Currently, if you pass `-C tracer=hw` you will only get code with DILabels if you enable debuginfo as well. This isn't ideal.~~

Currently, when using `-C tracer=hw` we implicitly enable debuginfo. If possible, it would be better if we could find a way to build the necessary debug contexts but without emitting regular debug info (e.g. by using some special debuginfo mode that only adds our labels).

**Comment by ptersilie** [comment]
We've implemented 1. in #52. @vext01 Should we close this issue, or leave it open as a reminder that ideally we'd want to do something more like 2.

**Comment by vext01** [comment]
I think the latter is supposed to read:
```
Make -C tracer=hw build the necessary debug contexts but without emitting regular debug info.
```

i.e. hw, not sw, right?

**Comment by vext01** [comment]
In fact, there are many references to sw, which should be hw I think.

**Comment by ptersilie** [comment]
Yes, that should be all `hw`, though that doesn't answer my question. ;-)

**Comment by vext01** [comment]
I think we can just edit this issue and cross out the stuff we have done already.
